ipn/ipnlocal/serve.go

```diff
@@ -219,6 +229,73 @@ func (b *LocalBackend) serveWebHandler(w http.ResponseWriter, r *http.Request) {
 	http.Error(w, "empty handler", 500)
 }
 
+func (b *LocalBackend) serveFileOrDirectory(w http.ResponseWriter, r *http.Request, fileOrDir, mountPoint string) {
+	fi, err := os.Stat(fileOrDir)
+	if err != nil {
+		if os.IsNotExist(err) {
+			http.NotFound(w, r)
```

Comment:
missing return?

Reply:
Good catch. Happens to be accidentally harmless in this case, but wrong nonetheless.

Reply:
PTAL